FiveCalls/FiveCalls/LocationSheet.swift

```diff
@@ -106,7 +106,9 @@ struct LocationSheet: View {
     
     func locationSearch() {
         _ = NewUserLocation(address: locationText) { loc in
-            delegate?.setLocation(location: loc)
+            DispatchQueue.main.async {
+                setLocation(loc)
```

Comment:
Hmm, I would have thought declaring `MainActor` above would have avoided needing this. Maybe putting the entire `AppState` object within `MainActor` would do the trick?